src/modules/handleInputOutput.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const readline = require('readline');
+
+const handleInputOutput = () => {
+  return new Promise(resolve => {
+    const message = 'Insert 4  different digits: ';
+
+    const terminal = readline.createInterface({
+      input: process.stdin,
+      output: process.stdout,
+    });
+
+    // eslint-disable-next-line no-console
+    console.log(message);
+
+    terminal.on('line', (UserNumber) => {
```

Comment:
Small question, why `UserNumber` starts from the Big letter?

Reply:
just for no reason, you know )))